### DISCUSSION TITLE
Inquiry About Hurricane Data Retrieval in R2D

### DISCUSSION THREAD

**Discussion by sejin0991** [NHERI-SimCenter/SimCenterCommon#307] [R2D] [answered]
Hi! 

While using R2D to assess community damage from hurricanes, it seems that users are required to provide hurricane data manually. In earlier versions, I recall there being an option to access an online database and retrieve historical hurricane data directly. I was wondering if this feature to fetch historical hurricane data has been removed.  
Thanks!

**Comment by jinyan1214** [comment, marked as answer]
Hello,

Thanks for your interest in using R2D! The hurricane database and historical hurricane data are still available in R2D. They are located in the "Tools" dropdown menu at the top of the R2D user interface. [Here](https://nheri-simcenter.github.io/R2D-Documentation/common/user_manual/usage/desktop/R2DTool/tools.html#hurricane-scenario) is some documentation about the tool. I hope you find the tool useful and please let us know if you have any further questions. 

Best,

Jinyan
